src/main/scala/scalismo/mesh/MeshOperations.scala

```diff
@@ -178,5 +179,27 @@ class TriangleMesh3DOperations(private val mesh: TriangleMesh3D) {
   def compact: MeshCompactifier = {
     mask(_ => true, _ => true)
   }
+
+  /**
+   * Attempts to reduce the number of vertices of a mesh to the given number of vertices.
+   *
+   * @param targetedNumberOfVertices The targeted number of vertices. Note that it is not guaranteed
+   *                                 that this number is reached exactly
+   * @return The decimated mesh
+   */
+  def decimate(targetedNumberOfVertices: Int): TriangleMesh[_3D] = {
+    val refVtk = MeshConversion.meshToVtkPolyData(mesh)
+    val decimatePro = new vtk.vtkDecimatePro()
```

Comment:
I am not qualified to judge the different mesh reductions implemented in vtk. So I rely on your choice here.